.github/workflows/main.yml

```
@@ -65,9 +65,21 @@ jobs:
         with:
           path: 'packages/site/dist'
 
-      # Publish to Chromatic (from any push)
+      # Publish to Chromatic (from any push *except* main)
       - name: Publish to Chromatic
-        if: github.event_name == 'push'
+        if: github.event_name == 'push' && !startsWith(github.ref, 'refs/heads/main')
+        uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          onlyChanged: true # Use TurboSnap for PR builds
```

Is `onlyChanged` the only difference between the `main`/non-`main` configurations? If so then would something like this work to reduce duplication?

```suggestion
          onlyChanged: !startsWith(github.ref, 'refs/heads/main') # Use TurboSnap for PR builds
```